app/controllers/container_controller.rb

```diff
@@ -73,7 +73,15 @@ def show_timeline
   end
 
   def explorer
+    rec_cls = "container"
```

Comment:
it is used in the url (#L82). I tried to follow the structure from vm_common (https://github.com/ManageIQ/manageiq/blob/master/app/controllers/vm_common.rb#L204), but I drop the local variable and just insert it as a string in the url.